src/pages/binnentuin/menu/Menu.jsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+
+import './Menu.scss';
+import logo_white from '../../../assets/img/logo_white.png';
```

Comment:
Deze regels is overbodig geworden en zou je eventueel nog kunnen verwijderen.

Reply:
done